addons/web/static/src/views/fields/properties/property_definition_selection.js

```diff
@@ -32,6 +33,36 @@ export class PropertyDefinitionSelection extends Component {
                 inputs[this.state.newOptionIndex].focus();
             }
         });
+
+        useSortable({
+            enable: () => {
+                if (!this.props.canChangeDefinition || this.props.readonly) {
+                    return false;
+                }
+
+                const element = document.activeElement;
+                if (
+                    element &&
+                    element.tagName === "INPUT" &&
+                    element.parentNode.classList.contains("o_field_property_selection_option") &&
+                    (!element.value || !element.value.length)
+                ) {
+                    // an empty option has the focus, and so it will be removed when
+                    // we will start moving, so we block the drag and drop in that case
+                    return false;
+                }
+                return true;
+            },
+            ref: this.propertyDefinitionSelectionRef,
+            handle: ".o_field_property_selection_drag",
+            elements: ".o_field_property_selection_option",
+            cursor: "grabbing",
+            onDrop: async ({ element, previous }) => {
+                const from = element.getAttribute("option");
+                const to = previous && previous.getAttribute("option");
```

Comment:
Would renamed "from" and "to" to "movedOption" and "destinationOption".

"from" and "to" would indicate indexes, here from what I understand "from" is the option being moved and "to" is the option after which we want it to go.